src/App.js

```diff
@@ -1,28 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Calculator />
```

Comment:
- [ ] Great job so far @zamcham You did an amazing job creating the calculator component. However, per the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md), you should :+1: 

> Create at least 1 child component
- [ ] This means `Calculator.js` should have at least 1 child component. :wink: So kindly implement this requirement as well :100: